Store/StoreAnalyticsOrderTracker.php

```diff
@@ -66,6 +69,27 @@ public function getFacebookPixelCommands()
 	}
 
 	// }}}
+	// {{{ public function getBingUETCommands()
+
+	public function getBingUETCommands()
+	{
+		$command = array(
+			'ec' => 'conversion',
+			'ea' => 'purchase',
+			'gv' => $this->getOrderTotal(),
+		);
```

Comment:
I think you are setting up these commands incorrectly. It should be as follows:
```
$command = array(
  array('ec', 'conversion'),
  array('ea', 'purchase'),
  array('gv', $this->getOrderTotal()),
);
```

Setting up the commands like this will generate the following output:
```
window.uetq.push({ "ec": "conversion" }, { "ea": "purchase" }, { "gv": "395.00"});
```

Right now we get the following:
```
window.uetq.push("conversion", "purchase", "395.00");
```